cfme/services/catalogs/catalog.py

```diff
@@ -151,7 +151,7 @@ def create(self, name, description, items=None):
         view.add_button.click()
         catalog = self.instantiate(name=name, description=description, items=items)
         view = self.create_view(CatalogsView)
-        assert view.is_displayed
```

Comment:
you don't need assert since wait_displayed will fail otherwise.

Reply:
@niyazRedhat even I would not use `view.wait_displayed()` as in last step only you are creating `view`.
best way:
```
view = self.create_view(CatalogsView, wait="10s")
```